load-tests/h1_transformation/src/h1_transformation.bal

```diff
@@ -18,56 +18,58 @@ import ballerina/http;
 import ballerina/log;
 import ballerina/xmldata;
 
-listener http:Listener securedEP = new(9090, {
-    secureSocket:  {
+listener http:Listener securedEP = new(9090,
+    httpVersion = "1.1",
+    secureSocket = {
         key: {
             path: "./security/ballerinaKeystore.p12",
             password: "ballerina"
         }
     }
-});
+);
 
-final http:Client nettyEP = check new("https://netty:8688", {
-    secureSocket: {
+final http:Client nettyEP = check new("https://localhost:8688",
+    httpVersion = "1.1",
+    secureSocket = {
         cert: {
             path: "./security/ballerinaTruststore.p12",
             password: "ballerina"
         },
         verifyHostName: false
     }
-});
+);
 
 service /transform on securedEP {
-    resource function post .(http:Caller caller, http:Request req) {
+    resource function post .(http:Caller caller, http:Request req) returns error? {
         json|error payload = req.getJsonPayload();
-        if (payload is json) {
+        if payload is json {
             xml|xmldata:Error? xmlPayload = xmldata:fromJson(payload);
-            if (xmlPayload is xml) {
+            if xmlPayload is xml {
                 http:Request clinetreq = new;
                 clinetreq.setXmlPayload(xmlPayload);
                 http:Response|http:ClientError response = nettyEP->post("/service/EchoService", clinetreq);
-                if (response is http:Response) {
-                    error? result = caller->respond(response);
+                if response is http:Response {
+                    return caller->respond(response);
```

Comment:
Here also, simply return http:Response

Reply:
Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e) and [3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684)